geomesa-quickstart-kafka/README.md

```diff
@@ -21,7 +21,7 @@ Prerequisites
 -------------
 
 * basic knowledge of [GeoTools](http://www.geotools.org), [GeoServer](http://geoserver.org), and Kafka
-* an instance of Kafka 0.8.2.x with (an) appropriate Zookeeper instance(s)
+* an instance of Kafka 0.8.2.x, 0.9.0.1, or 0.10.0.1 with (an) appropriate Zookeeper instance(s)
```

Comment:
Maybe we could add a paragraph above or below here just talking about the fact that we have three slightly different distributions of the tutorial code that depends on Kafka version and that (for now) you need profiles (see below) to build Kafka 0.9 or Kafka 0.10. You will also have to build the corresponding GeoMesa code with its profiles as well.